### DISCUSSION TITLE
Change image inside a scroll

### DISCUSSION THREAD

**Discussion by fsemecurbe** [qmd-lab/closeread#124] [General]
First at all, it's a very nice work ! I have a question how I can change the image inside a scroll section ?

**Comment by jimjam-slam** [comment]
Hey @fsemecurbe, thanks for hopping in! This demo is a good example of switching from one image to another in a scroll section:

Page: https://closeread.dev/gallery/demos/build-up-plots/
Code: https://github.com/qmd-lab/closeread/blob/main/docs/gallery/demos/build-up-plots/index.qmd

In your section you'll need two images (in this demo they're R plots, but you can use pre-made Markdown ) surrounded by Pandoc divs, and they need IDs that start with `#cr-`:

```
:::{#cr-image1}
(make image here)
:::

:::{#cr-image2}
(make image here)
:::

```

Then you need two _triggers_: narrative content that will make them appear. You make triggers by referencing these image IDs using Quarto cross-references:

```
Let's show the first image @cr-image1

Now we'll show the second image! @cr-image2
```

Give it a try and let us know how you go! 🥳

**Reply by fsemecurbe** [reply]
thank you @jimjam-slam ! I ve worked, for agricultural statistic  in France and we have made https://vizagreste.agriculture.gouv.fr/performances-econonomiques-industries-agroalimentaires.html with svelte. But it's too difficult for a statisician to use it . Closeread is a game changer, I think we can use it to make new dataviz ! thank you so much

**Reply by andrewpbray** [reply]
This is beautiful work. Can't wait to see what you put together with Closeread!
